content/en/getting_started/synthetics/browser_test.md

```diff
@@ -28,12 +28,13 @@ In this example, a browser test is configured to map a user's journey from addin
 3. Select **Browser Test**.
 4. Define the configuration of your browser test:
 
-    - Add the URL of the website you’re interested in monitoring. If you don’t know what to start with, you can use `https://www.shopist.io` as a test web application.
-    - Name the test.
-    - You can set tags such as `prod` and `shopist` on your test. Tags will help to keep things organized and allow you to quickly find the tests you're interested in on the homepage.
-    - Choose devices and locations for testing. In this example, the test is only run on **Large Laptops** and on **English speaking countries**.
-    - Specify a test frequency.
-    - Set alert conditions to determine the circumstances under which you want a test to send a notification alert.
+    - **Starting URL**: Add the URL of the website you’re interested in monitoring. If you don’t know what to start with, you can use `https://www.shopist.io` as a test web application.
+    - **Name**: Name the test.
+    - **Tags**: You can set tags such as `prod` and `shopist` on your test. Tags will help to keep things organized and allow you to quickly find the tests you're interested in on the homepage.
+    - **Browsers & Devices**: Choose the devices you want to use for testing. In this example, the test is only run on **Large Laptops**.
+    - **Locations**: Choose one of the **managed locations** to run your test from. In this example, the test is run in **Americas and Europe**.
+    - **Specify a test frequency**: Select how often you would like the test to run.
+    - **Alert Conditions**: Set alert conditions to determine the circumstances under which you want a test to send a notification alert.
```

Comment:
Let's put fast retries first (we'll do that in the UI soon) and remove the reference about fast retries interval as well. We could even only focus on one of these options in each getting started (fast retries in browser tests and alerting rule in API test to make that section more readable).